flow-typed/platform.js

```diff
@@ -58,15 +58,33 @@ declare type FeatureFlags = { [string]: boolean | string };
 
 declare type Params = { [string]: string };
 
+type CookieOpts = {|
+	value: string,
+	ttl?: number | null, // milliseconds
+	isHttpOnly?: boolean,
+	isSecure?: boolean,
+	isSameSite?: false | 'Strict' | 'Lax',
```

Comment:
Might be good to comment what the different options here mean

Reply:
This is sort of intentionally undocumented because it's only here to prevent Flow from yelling, but actually I think I can remove it again.